components/centraldashboard/public/components/main-page.js

```diff
@@ -28,6 +28,7 @@ import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
 
 import css from './main-page.css';
 import template from './main-page.pug';
+import logo from '../assets/kf-logo_64px.svg';
```

Comment:
Awesome! Though this change could have been in a separate PR?

Reply:
Actually I changed it because it made for easier testability. Otherwise, I would have had to mock window.fetch and make it resolve a promise with an object with a .text() method which would resolve to something that could be injected into the DOM. This seemed like an easier path and it eliminated the request when deployed.